app/components/display/taxonomy_card_component/taxonomy_card_component.html.haml

```diff
@@ -11,9 +11,12 @@
   .description
     = render TextAreaFieldComponent.new(value: @taxonomy.description)
   .ontologies-cards
-    - @taxonomy.ontologies.each do |ontology|
+    - @taxonomy.ontologies.each_with_index do |ontology, index|
```

Comment:
this is not related to this PR

Reply:
It happened because of the branch change it was master then changed it to dev